package/gluon-mesh-vpn-wireguard/files/usr/bin/wgcheck

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+curtime=$(date +%s)
+
+get_wg_interfaces() {
+  ubus -S call network.interface dump | jsonfilter -e '@.interface[@.up=true && @.proto="gluon_wireguard"].l3_device'
+}
+
+get_connection_count() {
+  ubus -S call network.interface dump | jsonfilter -e '@.interface[@.up=true && @.proto="gluon_wireguard" && @].l3_device' | wc -l
+}
+
+get_interface_from_ifname() {
+  ubus -S call network.interface dump | jsonfilter -e "@.interface[@.proto=\"gluon_wireguard\" && @.l3_device=\"$1\"].interface"
+}
+
+# purge wg interface that have terminated
+for i in $(get_wg_interfaces)
+do
+  line=$(wg show $i latest-handshakes)
+  if [[ -n "${line}" ]]; then
+    latest=$(echo ${line}| awk '{print $2}')
+    diff=$((curtime-latest))
+    if [[ $diff -gt 600 ]]; then
+      ifdown $(get_interface_from_ifname ${i})
+    fi
+  else
+    ifdown $(get_interface_from_ifname ${i})
+  fi
+done
+
+# in case less than our peer-limit connections is "up", start all wg interfaces that are currently down
+if [[ "$(uci get wireguard.mesh_vpn_backbone.enabled)" == "1" ]] &&
```

Comment:
```suggestion
if [[ "$(uci get wireguard.mesh_vpn.enabled)" == "1" ]] &&
```

Reply:
but everywhere: https://code.bremen.freifunk.net/ffhb/firmware/gluon/commit/df4d76204cb8b541f42afe0ba1195de83a192c13

Reply:
There is a mesh-vpn that allows to connect to other nodes on the network via vpn instead of wifi. And there is a backbone peer group that is comprised of the gateways of that community. There may be other peer groups providing mesh-vpn. The naming is fine.

Reply:
So maybe you want to check both is enabled?